liiweb/views/legislation.py

```diff
@@ -87,7 +87,7 @@ def add_children(self, queryset):
         )
 
         children = defaultdict(list)
-        children_qs = self.get_base_queryset().filter(
+        children_qs = Legislation.objects.filter(
```

Comment:
Also add in

```python
children_qs.preferred_language(get_language(self.request))
```

so that the best language is used where possible.

Reply:
right, added